tests/wpt/tests/css/css-grid/grid-items/grid-items-relative-positioned-containing-block.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Grid Layout Test: Relative Positioned Grid Items as a Containing Block of Absolute positioned descendant</title>
+<link rel="author" title="Jo Steven Novaryo" href="mailto:steven.novaryo@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-position-3/#absolute-cb">
+<link rel="help" href="https://drafts.csswg.org/css-position-3/#fixed-cb">
+<link rel="help" href="https://drafts.csswg.org/css-grid/#grid-items">
+<link rel="help" href="https://github.com/servo/servo/issues/34535">
+<link rel="match" href="../reference/grid-items-relative-positioned-containing-block-ref.html">
```

Comment:
This is a bit dangerously tall for a reftest. Reftests render in a 800x600 viewport, and I'm getting a 555px document height here, very close to the 600px limit.

Consider splitting this into multiple files, you can e.g. use the same name with `-001`, `-002`, etc. suffixes. And if you put each green square into its own file, you can use an existing reference like `/css/reference/ref-filled-green-100px-square.xht`